scripts/manageLAND.js

```diff
@@ -173,7 +295,22 @@ async function transferLANDs(args) {
   if (txsToRetry.length > 0 && shouldRetry) {
     const landBatchessToRetry = txsToRetry.map(tx => tx.data)
     log.info(`Retrying on ${landBatchessToRetry.length} recipients`)
-    return transferLANDs(landBatchessToRetry, ...args.slice(1))
+    return transferManyLAND(landBatchessToRetry, ...args.slice(1))
+  }
+}
+
+async function checkAllowance(land) {
+  const account = eth.getAccount()
+  const LANDRegistryContract = eth.getContract('LANDRegistry')
+  if (
+    (await LANDRegistryContract.getApproved(land.id)).toLowerCase() !==
```

Comment:
Maybe we can move the `await` to a line above this one? to make it a bit less convoluted

Reply:
Not for this PR, but it'd be awesome to have a `.isSameAddress()` method somewhere too.
For the future™